ruby-tapas-downloader.rb

```diff
@@ -35,12 +35,15 @@ def get_episode_links
   def download_missing_episodes
     yield if block_given?
 
+    browser = LoggedBrowser.new.call
```

Comment:
Also, another thought since we are just going to have a single instance of this we could just make this class a singleton and then we don't have to mess with passing the instance around,

What do you think?